### ISSUE TITLE
clear data

### ISSUE THREAD

**Issue by alicera** [ultralytics/yolov5#6332]
### Search before asking

- [X] I have searched the YOLOv5 [issues](https://github.com/ultralytics/yolov5/issues) and [discussions](https://github.com/ultralytics/yolov5/discussions) and found no similar questions.

### Question

Do you have the method to clear the training dataset ?
For example, error label cause the loss

**Comment by glenn-jocher** [comment]
@alicera I don't understand the question.
